modules/coreos-layering-configuring-on-remove.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/coreos-layering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coreos-layering-configuring-on-remove_{context}"]
+= Removing an on-cluster custom layered image
+
+To prevent the custom layered images from taking up excessive space in your registry, you can automatically remove an on-cluster custom layered image from the repository by deleting the `MachineOSBuild` object that created the image.
+
+The credentials provided by the registry push secret that you added to the `MachineOSBuild` object must grant the permission for deleting an image from the registry. If the delete permission is not provided, the image is not removed when you delete the `MachineOSBuild` object.
+
+Note that the custom layered image is not deleted if the image is either currently in use on a node or is desired by the nodes, as indicated by the `machineconfiguration.openshift.io/currentConfig` or `machineconfiguration.openshift.io/desiredConfig` annotation on the node.
```

Comment:
The annotations for the image here is `machineconfiguration.openshift.io/currentImage` and `machineconfiguration.openshift.io/desiredImage`.

Reply:
@umohnani8 Is this also the case in 4.19+? I would need to update the annotation in those branches also.

Reply:
@mburke5678 yes, that is the case in 4.19 and later as well